refactor: Apply Wikipedia's style of bolding to multiple pages (#2662)
privacyguides/privacyguides.org: docs/email-clients.md

description: These email clients are privacy-respecting and support OpenPGP email encryption.
cover: email-clients.webp
---

> The list of recommendations here contains **email clients** that support both [OpenPGP](encryption.md#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](basics/multi-factor-authentication.md) and prevent account theft.

Reviewer: Feels wordy

```suggestion
The **email clients** we recommend support both [OpenPGP](encryption.md#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](basics/multi-factor-authentication.md) and prevent account theft.
```
